src/utils/helpers/date-and-time-formatter.ts

```diff
@@ -0,0 +1,37 @@
+const getDate = (dateParam: Date | string): string => {
+  if (!dateParam) {
+    console.error('getDate() failed');
+    return '';
+  }
+
+  const date = new Date(dateParam);
+  const nowDate = Date.now();
+  const diff = nowDate - date.getTime();
+
+  const sec = 1000;
+  const min = 60 * sec;
+  const hour = 60 * min;
+  const day = 24 * hour;
+
+  const isLessThanAMin = Math.floor(diff / min) <= 0.0;
+  const isLessThanAnHour = Math.floor(diff / hour) <= 0.0;
+  const isLessThanADay = Math.floor(diff / day) <= 0.0;
+  const isLessThan48H = Math.floor(diff / (2 * day)) <= 0.0;
```

Comment:
Kan forenkles, right?
```suggestion
  const isLessThanAMin = diff < min;
  const isLessThanAnHour = diff < hour;
  const isLessThanADay = diff < day;
  const isLessThan48H = diff < 2 * day;
```

Reply:
Tror `Math.floor()` skal være der, men **0.0** skal erstattes med **0**, sånn at det bli helltall sammenligning